src/main/java/com/rultor/agents/req/Decrypt.java

```diff
@@ -58,11 +67,33 @@
     private final transient Profile profile;
 
     /**
+     * Proxy usage command string.
+     */
+    private final transient String proxy;
+
+    /**
      * Ctor.
      * @param prof Profile
+     * @param settings Proxy settings container
      */
-    Decrypt(final Profile prof) {
+    public Decrypt(final Profile prof, final Pair<String, Integer> settings) {
```

Comment:
this `public` modifier is redundant, since the class is visible only inside the package